File: dart/2543.dart
import 'dart:io';

void main() {
  while (true) {
    var input = stdin.readLineSync();

    if (input == null) {
      break;
    }

    String id = input.split(" ")[1];
    int count = 0;

    for (int i = 0; i < int.parse(input.split(" ")[0]); i++) {
      List<String> gp = stdin.readLineSync()!.split(" ");

      if (gp[0] == id && gp[1] == "0") {
        count++;
      }
    }

    print(count);
  }
}

File: python/2543.py
while True:
    try:
        n, id = input().split()
        cs = 0

        for i in range(int(n)):
            student, game = input().split()
            if student == id and game == "0":
                cs += 1

        print(cs)
    except EOFError:
        break
